CHANGELOG.md

```diff
@@ -2,12 +2,16 @@
 
 ## UNRELEASED
 
+### Adds
+
+* It is now best practice to deliver namespaced i18n strings as JSON files in module-level subdirectories of `i18n/` named to match the namespace, e.g. `i18n/ourTeam` if the namespace is `ourTeam`. This allows base class modules to deliver phrases to any namespace without conflicting with those introduced at project level. The `i18n` option is now deprecated in favor of the new `i18n` module format section, which is only needed if `browser: true` must be specified for a namespace.
```

Comment:
This is a new feature, with an incidental deprecation that won't break anything in the 3.x major series, so I think it best belongs in Adds.

Reply:
why is this added in this branch?

Reply:
Oh, I see it replaced below. ✔️